apps/meteor/app/2fa/server/code/TOTPCheck.ts

```diff
@@ -20,6 +20,10 @@ export class TOTPCheck implements ICodeCheck {
 			return false;
 		}
 
+		if (!user.services?.totp?.secret) {
```

Comment:
please @RocketChat/backend not sure if returning is the best case, maybe throw an error?

Reply:
returning `false` is fine.. it means it cannot not be verified by this class.

Reply:
any other concern @ggazzo ?